pkg/package-server/provider/registry_test.go

```diff
@@ -55,7 +56,7 @@ func server() {
 		logrus.Fatal(err)
 	}
 
-	loader := sqlite.NewSQLLoaderForDirectory(load, "manifests")
+	loader := sqlite.NewSQLLoaderForDirectory(load, filepath.Join("testdata", "manifests"))
```

Comment:
We've been trying to use the `testdata` convention throughout operator-framework, so I updated it while I was here. I can remove it from the PR, but I don't think it's too risky to leave it.